fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -296,14 +297,24 @@ private int getChildrenLimit() {
     @DELETE
     @Timed
     public Response deleteObject() {
-        evaluateRequestPreconditions(request, servletResponse, resource(), session);
+        FedoraResource resource = resource();
```

Comment:
Just a note on storing resource() in a variable for this method. This change was not strictly speaking in scope for this ticket, but it seemed like an easy and reasonable small gain in efficiency not to call resource() three times, when once would do :-)